### ISSUE TITLE
Add a way to toggle normal tree-view

### ISSUE THREAD

**Issue by dsandstrom** [olmokramer/atom-autohide-tree-view#3]
I like it so far, it's great that it doesn't shift my panes over when viewing the directories.

Maybe you can add a button to toggle the normal tree-view. So I can lock the tree-view in place.

Or maybe change the behavior so the normal tree-view shows up.  But, when the normal tree-view is hidden, your autohide-tree-view shows up on mouse over.

**Comment by olmokramer** [comment]
I wanted to do this, but for the sake of simplicity I left it out. I'll try to implement this somewhere this weekend

**Comment by olmokramer** [comment]
I just released `v0.11.0`. It registers a command - `autohide-tree-view:toggle-enabled` - that enables/disables autohide behavior. I did not add a keybinding for it by default, but you can do that in your `keymap.cson`, for example:

``` cson
'atom-workspace':
  'ctrl-\\': 'autohide-tree-view:toggle-enabled'
```

By default the <kbd>Ctrl-\</kbd> is used to show/hide the tree view.
